gobblin-docs/user-guide/Docker-Integration.md

```diff
@@ -18,68 +18,68 @@ The `gobblin/gobblin-wikipedia` repository contains images that run the Gobblin
 
-The Docker images for this repository can be found on Docker Hub [here](https://hub.docker.com/r/gobblin/gobblin-wikipedia/). These images are mainly meant to act as a "Hello World" example for the Gobblin-Docker integration, and to provide a sanity check to see if the Gobblin-Docker integration is working on a given machine. The image contains the Gobblin configuration files to run the [Gobblin Wikipedia job](../Getting-Started). When a container is launched using the `gobblin-wikipedia` image, Gobblin starts up, runs the Wikipedia example, and then exits.
+### Set working directory
 
-Running the `gobblin-wikipedia` image requires taking following steps (lets assume we want to an Ubuntu based image):
+Before running docker containers, set a working directory for Gobblin jobs:
 
-* Download the images from the `gobblin/gobblin-wikipedia` repository
+`export LOCAL_JOB_DIR=<local_gobblin_directory>`
 
-```
-docker pull gobblin/gobblin-wikipedia:ubuntu-gobblin-latest
-```
+We will use this directory as the [volume](https://docs.docker.com/storage/volumes/) for Gobblin jobs and outputs. Make sure your Docker has the [access](https://docs.docker.com/docker-for-mac/#file-sharing) to this folder. This is the prerequisite for all following example jobs.
 
-* Run the `gobblin/gobblin-wikipedia:ubuntu-gobblin-latest` image in a Docker container
+### Run the docker image with simple wikipedia jobs
 
-```
-docker run gobblin/gobblin-wikipedia:ubuntu-gobblin-latest
-```
+Run these commands to start the docker image:
 
-The logs are printed to the console, and no errors should pop up. This should provide a nice sanity check to ensure that everything is working as expected. The output of the job will be written to a directory inside the container. When the container exits that data will be lost. In order to preserve the output of the job, continue to the next step.
+`docker pull gobblin/gobblin-standalone:latest`
```

Comment:
We started using apache's repository for our images. Could we change this line to pull from https://hub.docker.com/r/apache/gobblin/tags?page=1&ordering=last_updated ?